src/document-templates/defaultTextResources.ts

```diff
@@ -59,7 +60,8 @@ export const defaultTextResources: Record<Language, Record<string, string>> = {
         'rental-agreement.title': 'Hyresavtal',
         'rental-agreement.legal-note.title': 'Hyresvillkor',
         'rental-agreement.legal-note.content':
-            'Utrustningen skall lämnas tillbaka i det skick och nedpackat på samma vis som den hämtades i. Detta innebär att alla kablar skall rullas ihop snyggt och om något blivit smutsigt, kladdigt eller lämnats kvar tejprester på skall detta göras rent.',
+            'Genom att signera detta avtal godkänner Hyrestagaren villkoren som återfinns på',
+        'rental-agreement.legal-note.url': 'www.rneventteknik.se/terms',
```

Comment:
Nice! I realize it probably makes no difference in practice since most browsers/pdf editors are very flexible, but my instinct would be to have separate fields for the url and shown text for the link, which would allow us to include "https://" in the actual link. Was it a deliberate choice to use the same text resource for both url and shown text?

Reply:
I thought briefly about it and decided on this for simplicity but I can add a separate field. A bit strange to have it among the text resources but that's maybe fine...?

Reply:
I think it is fine, it is still part of the text content